GTBitsOfGood/angels-among-us
# Onboarding Scaffolding

## Onboarding Scaffolding

Issue Number(s): #17 

Adds onboarding scaffolding structure that will automatically generate question pages with progress bar, question number, question title, question description, and options (not entirely functional) given a JS object that is in the page code. 

### Checklist


### Critical Changes

- None

### Related PRs

- None

### Testing

Test layout on various device types:
- Try out different dimensions when viewing the webpage (iPhone, iPad, desktop, etc) and make sure that the layout looks consistent and reasonable.

Test dynamic generation of questions:
- Add/remove questions and ensure that progress bar and number of questions is updated and that the new question appears in its correct place in the question sequence or that the removed question disappears
